lib/sanbase_web/graphql/resolvers/billing_resolver.ex

```diff
@@ -174,6 +174,10 @@ defmodule SanbaseWeb.Graphql.Resolvers.BillingResolver do
 
   defp handle_subscription_error_result(result, log_message, params) do
     case result do
+      {:error, %Stripe.Error{message: message} = reason} ->
+        Logger.error("#{log_message} - reason: #{inspect(reason)}")
```

Comment:
To match other reason reports:
```suggestion
        Logger.error("#{log_message}. Reason: #{inspect(reason)}")
```